python/src/trezorlib/messages.py

```
@@ -5152,6 +5153,20 @@ def __init__(
         self.pre_firmware = pre_firmware
 
 
+class WriteSEPrivateKey(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = 10027
+    FIELDS = {
+        1: protobuf.Field("private_key", "bytes", repeated=False, required=True),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)